astroquery/ipac/irsa/irsa_dust/tests/test_irsa_dust.py

```diff
@@ -117,21 +113,20 @@ def test_xml_err(self):
 # TODO : Add more examples. Add for "1 degree"-like parameters
     @pytest.mark.parametrize(
         ('coordinate', 'radius', 'expected_payload'),
-        [(galcoords["m81"], None, dict(locstr=format(galcoords['m81']))),
-         (galcoords["m31"], "5d0m", dict(locstr=format(galcoords['m31']),
-                                         regSize=5.0)),
-         (galcoords["m31"], 5 * u.deg, dict(locstr=format(galcoords['m31']),
-                                            regSize=5)),
-         ("m31", 5 * u.deg, dict(locstr='m31', regSize=5)),
-         (coordinates.SkyCoord(ra=148.888221083 * u.deg,
-                               dec=69.065294722 * u.deg, frame='icrs'),
-          5 * u.deg,
-          {'locstr': format(coordinates.SkyCoord(ra=148.888221083 * u.deg,
-                                                 dec=69.065294722 * u.deg,
-                                                 frame='icrs')),
-           'regSize': 5}), ])
+        (
```

Comment:
again, don't use black, please reformat without empty lines with single parens and trailing comas